docs/pages/identity-governance/access-lists/terraform.mdx

```diff
@@ -0,0 +1,314 @@
+---
+title: Managing Access Lists with Terraform
+description: Learn how to manage Access Lists and their members with Terraform.
+labels:
+ - how-to
+ - identity-governance
+---
+
+Is is possible to manage Access List and their members using Terraform. This guide will help you:
+
+- Understand how to manage Access Lists with Terraform
+- Familiarize with basic Access List types
+- Understand how to manage Access List members, and which Access List types allow that and why
+
+
+## How it works
+
+The goal here is to create a simple organization structure represented with Access Lists using
+Terraform. It will consist of:
+
+- Access List representing a team
+- Access List representing access to a resource (staging database)
+- assigning the team access to the resource
+- (optionally) assigning Okta group access to the resource
+
+Access List management with Terraform is mostly done with 2 Terraform resources
+`teleport_access_list` and `teleport_access_list_member`.
+
+There are some nuances to members management. This is where concept of *Access List type* comes
+into play. There are 2 key Access List types *default* and *static*.
+
+**default**
```

Comment:
I would use a heading here since this is breaking up the document into subsections